packages/core/src/domain/configuration.ts

```diff
@@ -75,7 +75,7 @@ export type Configuration = typeof DEFAULT_CONFIGURATION &
     cookieOptions: CookieOptions
 
     service?: string
-    beforeSend?: (event: any) => void
+    beforeSend?: (event: any) => unknown
```

Comment:
this type is used internally, it could be nice to update signatures exposed to customer to add the `| boolean` return type

Reply:
Done